resources/views/livewire/library-resources.blade.php

```diff
@@ -0,0 +1,190 @@
+                    <x-slot name="summary">{{ __('Area of impact') }}</x-slot>
+                    <fieldset class="filter__options field @error('status') field--error @enderror">
+                        <legend class="visually-hidden">{{ __('Area of impact') }}</legend>
+                        @foreach ($impactsData as $impact)
+                            <div class="field">
+                                <x-hearth-input id="impact-{{ $impact['value'] }}" name="impacts[]" type="checkbox"
+                                    value="{{ $impact['value'] }}" wire:model.live="impacts" />
+                                <label for="impact-{{ $impact['value'] }}">{{ $impact['label'] }}</label>
+                            </div>
+                        @endforeach
+                        <x-hearth-error for="status" />
+                    </fieldset>
+                </x-expander>
+                <x-expander :level="3">
+                    <x-slot name="summary">{{ __('Language') }}</x-slot>
+                    <fieldset class="filter__options field @error('status') field--error @enderror">
+                        <legend class="visually-hidden">{{ __('Language') }}</legend>
+                        @foreach ($languagesData as $language)
+                            <div class="field">
+                                <x-hearth-input id="language-{{ $language['value'] }}" name="languages[]"
+                                    type="checkbox" value="{{ $language['value'] }}" wire:model.live="languages" />
+                                <label for="language-{{ $language['value'] }}">{{ $language['label'] }}</label>
+                            </div>
+                        @endforeach
+                        <x-hearth-error for="status" />
+                    </fieldset>
+                </x-expander>
+                <div class="mt-6">
+                    <button class="secondary" type="button"
+                        wire:click="selectNone()">{{ __('Select none') }}</button>
+                </div>
+            </div>
+            <div class="md:pl-4">
+                <section aria-labelledby="collection-title">
+                    <div class="resources stack">
+                        @forelse($resources as $resource)
+                            <x-card.resource :model="$resource" :level="2" />
+                        @empty
+                            <p>{{ __('No resources found.') }}</p>
+                        @endforelse
+                    </div>
+                </section>
+
+                {{ $resources->onEachSide(2)->links('vendor.livewire.tailwind-custom') }}
+            </div>
+        </div>
+    </x-section>
+    @can('update', $library)
+        <p class="mt-12"><a class="cta secondary"
+                href="{{ route('filament.admin.resources.libraries.edit', $library) }}">@svg('heroicon-o-pencil', 'mr-1')
+                {{ __('Edit library') }}</a></p>
+    @endcan
```

Comment:
I think it was a good idea to add this link, but wonder if it should be at the top of the page; similar to how other similar edit options are in the UI. E.g. the public pages for an organization.